src/api/entities/confidential/__tests__/ConfidentialTransaction/index.ts

```diff
@@ -149,6 +152,85 @@ describe('ConfidentialTransaction class', () => {
     });
   });
 
+  describe('method: onStatusChange', () => {
+    let bigNumberToU64Spy: jest.SpyInstance;
+    let instructionStatusesMock: jest.Mock;
+    const rawId = dsMockUtils.createMockU64(new BigNumber(1));
```

Comment:
nit -> I prefer initializing in the beforeEach/beforeAll section especially the mock values